CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Updated to GraphDB [11.1.0](https://graphdb.ontotext.com/documentation/11.1/release-notes.html#graphdb-11-1-0)
 - Updated the `extraObjects` template processing to additionally support rendering string Helm templates.
+- Updated backup CronJob to use form data and removed deprecated backup-options json field.
```

Comment:
Rebase the branch on develop/12.2 and update the release version in the changelog